lib/rrd_updates.ml

```diff
@@ -192,6 +192,19 @@ let create_multi prefixandrrds start interval cfopt =
        (fun (_prefix,rrd) ->
           (* Find the rrds that satisfy the requirements *)
           Rrd.find_best_rras rrd pdp_interval cfopt start) prefixandrrds) in
+  let first_rra =
```

Comment:
this new definition doesn't preserve the old behaviour - i need to get the first rra from any list, rather than from only the first list